site/content/en/docs/administration/basics/installation.md

```diff
@@ -91,7 +93,10 @@ For access from China, read [sources for users from China](#sources-for-users-fr
   docker-compose up -d
   ```
 
-- Use `CVAT_VERSION` environment variable to specify the version of CVAT you want to install (e.g `v2.1.0`, `dev`):
+- (Optional) Use `CVAT_VERSION` environment variable to specify the version of CVAT you want to
```

Comment:
I don't like that we have these instructions several times. But it looks like many other steps are cloned as well.